easybuild/easyconfigs/g/GPyTorch/GPyTorch-1.3.0-foss-2020b.eb

```diff
@@ -8,7 +8,6 @@ description = "GPyTorch is a Gaussian process library implemented using PyTorch.
 
 toolchain = {'name': 'foss', 'version': '2020b'}
 
-source_urls = [PYPI_SOURCE]
```

Comment:
@Flamefire see an error downloading here:
```
== 2021-03-23 18:15:48,502 filetools.py:723 WARNING URL https://sources.easybuild.io/g/GPyTorch/gpytorch-1.3.0.tar.gz was not found (HTTP response code 404), not trying again
== 2021-03-23 18:15:48,504 filetools.py:749 WARNING Download of https://sources.easybuild.io/g/GPyTorch/gpytorch-1.3.0.tar.gz to /eb/maint/sources/g/GPyTorch/gpytorch-1.3.0.tar.gz failed, done trying
```

Reply:
@verdurin Are you using latest `develop` for easyblocks? See https://github.com/easybuilders/easybuild-easyblocks/pull/2364

Reply:
@boegel rebuilding with today's `develop` now.

Reply:
@verdurin Hope you don't actually rebuild 79ECs ;) I added my test cmd to the top: `eb --from-pr 12451 --rebuild --fetch --force-download --sourcepath=/dev/shm/sources --upload-test-report`

Reply:
My build failure doesn't look related to this PR.